diff --git a/node_modules/protobufjs/cli/targets/static.js b/node_modules/protobufjs/cli/targets/static.js
index d535e44..d4a061f 100644
--- a/node_modules/protobufjs/cli/targets/static.js
+++ b/node_modules/protobufjs/cli/targets/static.js
@@ -682,7 +682,10 @@ function buildEnum(ref, enm) {
         comment.push((config.forceEnumString ? "@property {string} " : "@property {number} ") + key + "=" + val + " " + (enm.comments[key] || key + " value"));
     });
     pushComment(comment);
-    push(escapeName(ref) + "." + escapeName(enm.name) + " = (function() {");
+    if (!ref && config.es6)
+        push("export const " + escapeName(enm.name) + " = " + escapeName(ref) + "." + escapeName(enm.name) + " = (() => {");
+    else
+        push(escapeName(ref) + "." + escapeName(enm.name) + " = (function() {");
     ++indent;
         push((config.es6 ? "const" : "var") + " valuesById = {}, values = Object.create(valuesById);");
         var aliased = [];
